infra/resources/modules/commons/apim.tf

```diff
@@ -52,6 +52,16 @@ module "apim" {
   tags = var.tags
 }
 
+resource "azurerm_key_vault_access_policy" "apim_policy" {
+  key_vault_id            = module.key_vault.id
```

Comment:
cc @christian-calabrese isn't this doable with the role module yet?

Reply:
it looks like the `azure_role_assignments` module do the job

Reply:
Thank you @gunzip, didn't notice the message! Happy to see that's going to be adopted @kin0992